libminifi/test/unit/MemoryUsageTest.cpp

```diff
@@ -20,22 +20,22 @@
 #include "utils/OsUtils.h"
 #include "../TestBase.h"
 
-TEST_CASE("Test memory usage", "[testmemoryusage]") {
+TEST_CASE("Test Physical memory usage", "[testphysicalmemoryusage]") {
   constexpr bool cout_enabled = true;
```

Comment:
These hardware/platform specific functions are hard to test (because they depend on a lot of things).
I added cout the these tests so one can manually check them (with 3rd party tools) that they return the correct data.
I also wanted to give a simple switch, if sometime we want to mute these tests.